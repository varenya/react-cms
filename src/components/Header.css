.home-menu{
    text-align: left;
    background:#2d3e50;
    padding:0.5em;
    box-shadow: 0 1px 1px rgba(0,0,0,0.5);
}

.home-menu .pure-menu-heading{
    color:#fff;
}
