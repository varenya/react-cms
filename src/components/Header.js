import React, {Component} from 'react';

import './Header.css'

class Header extends Component {
    render() {
        const {welcome} = this.props;
        return (
          <div className="header">
            <div className="home-menu pure-menu pure-menu-horizontal pure-menu-fixed">
              <a className="pure-menu-heading" href="#">{welcome}</a>
            </div>
          </div>
        );
    }
}

export default Header;
