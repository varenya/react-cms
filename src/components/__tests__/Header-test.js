import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../Header';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';

test('Check <Header /> is getting rendered properly without errors!', () => {
    const component = renderer.create(
        <MuiThemeProvider>
            <Header welcome={"test"}/>
        </MuiThemeProvider>
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
})
