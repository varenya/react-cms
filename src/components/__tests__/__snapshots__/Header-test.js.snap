exports[`test Check <Header /> is getting rendered properly without errors! 1`] = `
<div
  className="header">
  <div
    className="home-menu pure-menu pure-menu-horizontal pure-menu-fixed">
    <a
      className="pure-menu-heading"
      href="#">
      test
    </a>
  </div>
</div>
`;
